### PR TITLE
Added Cake.Recipe and Wyam

### PR DESCRIPTION
**Pull request by gep13**
Relates to #1 and #2

### CODE REVIEW

**Comment by gep13** [comment]
@agc93 let me know what you think of this.  Thanks!

**Comment by gep13** [comment]
@agc93 ping...

File: build.ps1 [hunk 1]
```diff
@@ -165,7 +160,7 @@ if(-Not $SkipToolPackageRestore.IsPresent) {
     }
 
     Write-Verbose -Message "Restoring tools from NuGet..."
-    $NuGetOutput = Invoke-Expression "&`"$NUGET_EXE`" install -ExcludeVersion -OutputDirectory `"$TOOLS_DIR`""
+    $NuGetOutput = Invoke-Expression "&`"$NUGET_EXE`" install -ExcludeVersion -PreRelease -OutputDirectory `"$TOOLS_DIR`" -Source https://www.myget.org/F/cake/api/v3/index.json"
```
Comment (agc93) [inline build.ps1:163]: Out of curiosity, why using the MyGet feed here?

File: docs/input/blog/new-release-0.4.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+Title: New Release - 0.1.0
```
Comment (agc93) [inline docs/input/blog/new-release-0.4.0.md]: Document is named `new-release-0.4.0.md` but this implies 0.1.0 release?

File: build.ps1 (hunk 1 above)
Comment (gep13) [reply build.ps1:163]: At the minute, there is still a little bit of flux with regard to the 0.3.0 release of Cake.Recipe.  Until that stops, I have been pulling the CI build from the MyGet feed.  Once settled, the use of the MyGet feed can be removed.

File: docs/input/blog/new-release-0.4.0.md (hunk 2 above)
Comment (gep13) [reply docs/input/blog/new-release-0.4.0.md]: ah, good catch.  Copy/pasta gone wrong.

**Pull request merged**